lib/codepoint.js

```diff
@@ -0,0 +1,19 @@
+const { ValidationError } = require('./errors')
+
+function isNumber(value) {
+  return value.match(/^((0x[\d[a-f]+)|(\d+))$/gi)
+}
+
+function parseCodePoint(codePoint, propName) {
```

Comment:
It is possible to consider parse value which ise recived from json outpu like '\\f101' ?
Or maybe is possible to get json for icon-font-generator with js unicode like "\uf101' ?

Reply:
I've updated the code to support CSS unicode escape codes. I think this should do what you ask for.

Reply:
@Hammie really thank You for that. This is exactly what i wont.